packages/forms-web-app/src/controllers/final-comment/comments-question.js

```diff
@@ -0,0 +1,38 @@
+const { VIEW } = require('../../lib/views');
+
+exports.getCommentsQuestion = async (req, res) => {
+	let hasComments = null;
+
+	if (
+		req.session &&
+		req.session.finalComment &&
+		Object.prototype.hasOwnProperty.call(req.session.finalComment, 'hasComments')
+	) {
+		hasComments = req.session.finalComment.hasComments;
+	}
+
+	res.render(VIEW.FINAL_COMMENT.COMMENTS_QUESTION, {
+		hasComments
+	});
+};
+
+exports.postCommentsQuestion = async (req, res) => {
+	const {
+		body,
+		body: { errors = {}, errorSummary = [] }
+	} = req;
+
+	if (Object.keys(errors).length > 0) {
+		return res.render(VIEW.FINAL_COMMENT.COMMENTS_QUESTION, {
+			errors,
```

Comment:
Would it make sense to pass in hasComments here as well? Not sure if we want to persist the user's choice in the event of errors

Reply:
looking at other radio button pages in the full appeal journey, I don't see the user choice being persisted (the typical error would be that they haven't made a choice).